packages/orchestration/src/exos/cosmos-orchestration-account.js

```diff
@@ -1106,6 +1114,45 @@ export const prepareCosmosOrchestrationAccountKit = (
             watch(E(this.facets.helper.owned()).executeEncodedTx(msgs, opts)),
           );
         },
+        /** @type {HostOf<NobleMethods['depositForBurn']>} */
+        depositForBurn(destination, amount) {
+          return asVow(() => {
+            trace('depositForBurn', destination, amount);
+            const { helper } = this.facets;
+            const { chainAddress } = this.state;
+
+            chainAddress.chainId.startsWith('noble') ||
+              Fail`'depositForBurn' not supported on chain: ${q(chainAddress.chainId)}`;
```

Comment:
This won't work for testnets, as noble's is `grand-1`. Seems like the `ecosystem` field we were talking about, or even something in addition to that, on `Chain` would be helpful here.

Reply:
I think we're going to have to ask the ChainHub for info about the address.